### PR TITLE
Mlcube download docker cli hints

### PR DESCRIPTION
**Pull request by VukW**
Hide hints from docker cli output when we do `docker pull`

### CODE REVIEW

File: cli/medperf/entities/cube.py [hunk 1]
```diff
@@ -248,7 +248,10 @@ def _set_image_hash_from_registry(self):
         cmd = f"mlcube --log-level {config.loglevel} inspect --mlcube={self.cube_path} --format=yaml"
         cmd += f" --platform={config.platform} --output-file {tmp_out_yaml}"
         logging.info(f"Running MLCube command: {cmd}")
-        with pexpect.spawn(cmd, timeout=config.mlcube_inspect_timeout) as proc:
+        cmd_env = {**os.environ}
```
Comment (hasan7n) [inline cli/medperf/entities/cube.py:251]: @VukW why are we passing os.environ?

Comment (VukW) [reply cli/medperf/entities/cube.py:251]: When we are not passing any `env`, child process inherits parent one. That is a standard practice and `subprocess` and any other process-spawning frameworks would have the same behavior concerning envs. I.e. passing `env=None` is totally equal to passing `env=os.environ`. 
- [ptyprocess code that is called under the hood](https://github.com/pexpect/ptyprocess/blob/master/ptyprocess/ptyprocess.py#L288)
- [os doc that describes the difference between used `execv` and `execvpe`](https://docs.python.org/3/library/os.html#os.execvpe)

So, previously we actually passed parent's `env` to child process. And now we want to modify it a bit: add new env for docker engine. Thus, I copy our env and add a new kv pair if necessary

Comment (VukW) [reply cli/medperf/entities/cube.py:251]: * the whole PR is about adding `DOCKER_CLI_HINTS` env var to child process if docker engine is used

**Comment by VukW** [comment]
Replaced PR with new one to medperf main branch: https://github.com/mlcommons/medperf/pull/546

**Pull request closed without merging**